src/main/kotlin/lotto/domain/Lottos.kt

```diff
@@ -0,0 +1,9 @@
+package lotto.domain
+
+data class Lottos(val lottoList: List<Lotto>) : Iterable<Lotto> {
```

Comment:
코틀린은 언어 차원에서 [`Delegation`](https://kotlinlang.org/docs/delegation.html)을 지원합니다!
이를 활용하면 상위 타입의 함수를 오버라이딩하지 않고도 사용할 수 있어요.
```suggestion
data class Lottos(val lottoList: List<Lotto>) : List<Lotto> by lottoList
```

Reply:
delegate도 생각을 해봤는데 요것도 조금 고민이더라구요 ㅎㅎ..
일급컬렉션이라는게 특정 정보에 바로 접근할수 없도록 숨기고 여러가지 기능을 랩핑하기위해 쓴다고 생각하는데
리뷰어님이 예시 들어주신대로 위임을 하게 되면 특정 정보를 숨기는 일급컬렉션으로써의 장점 하나를 잃게 된다고 생각했는데
요것도 리뷰어님 의견이 궁금합니다~!